test/examples/test_examples.py

```diff
@@ -37,3 +39,35 @@ def test_basic_configuration():
         print('TIMEOUT')
 
     assert(ret)
+
+def test_hello_world():
+    """."""
+    ret = False
+    out = ''
+    with Popen('@DOCKER_EXECUTABLE@ compose -f hello_world.compose.yml up',
```

Comment:
I think we can use `run` instead, can't we?

Reply:
Performing other approach for this